Moved scores to keep consistency with playnite

BanCrash opened the pull request:
Hi, I moved the scores localization in the details pane in order to keep consistency with playnite itself, since currently the order was different between playnite and this theme. Feel free to reject this if you want to.

Playnite edition view: user - critics - community
![image](https://user-images.githubusercontent.com/11770760/171168139-d49913b7-51fb-41e6-9160-c6626a0ac052.png)

Theme before these changes: community - critics - user

Grid view:
![image](https://user-images.githubusercontent.com/11770760/171169286-efebc836-cc0a-4340-998f-4fd400f08cd2.png)

Details view:
![image](https://user-images.githubusercontent.com/11770760/171169350-4caece24-2933-4960-a6f1-44ce9d593896.png)

Theme after these changes: user - critics - community

Grid view:
![image](https://user-images.githubusercontent.com/11770760/171168697-990d2659-42c1-4efb-8da0-6feecd6c424e.png)

Details view:
![image](https://user-images.githubusercontent.com/11770760/171168804-dd5d9935-d2e2-4bfc-bec1-ecfc25e790e8.png)

MCC321-QC commented:
Another good idea! Thanks [BanCrash](https://github.com/BanCrash)

MCC321-QC reviewed (approved):
PART_ElemCriticScore should be before PART_ElemCommunityScore. Don't worry, im planning another update soon so I will include these change in the next update. Thanks for your contribution!

BanCrash commented:
> PART_ElemCriticScore should be before PART_ElemCommunityScore. Don't worry, im planning another update soon so I will include these change in the next update. Thanks for your contribution!

Ouch, I modified the file directly on Playnite folder and then pasted to the Github folder but it seems that I modified again and forgot to paste it. Sorry about that.
